blog/2022-07-15-typescript-1.0.0.md

```diff
@@ -0,0 +1,113 @@
+# 1.0.0 release of our TypeScript SDK
+
+After almost 2 years of development, 728 commits, 431 pull requests, and 48 releases we're finally releasing the stable
+version of the Temporal TypeScript SDK.
+
```

Comment:
```suggestion

<!--truncate-->

```

Reply:
What's this?

Reply:
https://docusaurus.io/docs/blog#blog-list